spec/Appendix C -- Schema Coordinates.md

```diff
@@ -0,0 +1,93 @@
+# Appendix: Schema Coordinates
+
+Schema Coordinates are human readable strings that uniquely identify an element defined in a GraphQL Schema.
+
+## Definition 
+
+SchemaCoordinates :
+  - TypeDefinitionName FieldSpecifier?
+  - EnumName EnumValueSpecifier?
+  - @ DirectiveName ArgumentSpecifier?
+  - UnionName
+
+TypeDefinitionName:
+  - ObjectTypeName
+  - InterfaceTypeName
+
+FieldSpecifier :
+  - . FieldName ArgumentSpecifier?
+
+ArgumentSpecifier :
+  - ( ArgumentName: )
+
+EnumValueSpecifier :
+  - . EnumValue
+
+## Examples
+
+This section shows example coordinates for the possible schema element types this syntax covers.
+
+All examples below will assume the following schema:
+
+```graphql example
+directive @private(scope: String!) on FIELD
+
+interface Address {
+    city: String
+}
+
+type User implements Address {
+    name: String
+    reviewCount: Int
+    friends: [User]
+    email: String @private(scope: 'loggedIn')
+    city: String
+}
+
+type Business implements Address {
+    name: String
+    address: String
+    rating: Int
+    city: String
+}
+
+union Entity = User | Business
+
+enum SearchFilter {
+    OPEN_NOW
+    DELIVERS_TAKEOUT
+    VEGETARIAN_MENU
+}
+
+type Query {
+    searchBusiness(name: String!, filter: SearchFilter): Business
+}
+```
```

Comment:
This example lacks custom scalars and input objects that would be useful to round it out.

Reply:
Hmm hadn't thought about custom scalars! To clarify, assuming `scalar DateTime` or something is defined, we want `DateTime` to be a valid schema coordinate?

I guess it makes sense that you may want to refer to a custom scalar (since they can have directives applied, or maybe you just want to track how often a custom scalar is being used)

I guess it raises the question - what about built in scalars? Is `String` a valid schema coordinate too lol?

(I think as an artifact, it would have to be - just presumably not a very useful artifact...)

I'll call it out as an example though.

Thanks!!!

Reply:
Absolutely, built in scalars should be included too :+1: